src/machines/chronograf-resources.json

```diff
@@ -33,13 +26,10 @@
                 "description": "Administrator password used when provisioning virtual machines"
             }
         },
-        "influxTags": {
+        "networkSettings": {
             "type": "object",
-            "defaultValue": {
-                "provider": "influxdata"
-            },
             "metadata": {
-                "description": "Unique identifiers to allow the Azure Infrastructure to understand the origin of resources deployed to Azure. You do not need to supply a value for this."
+                "description": "Network settings to deploy on"
```

Comment:
Was there some text cut off here?

Reply:
hey @gunnaraasen -- good catch, yup i accidentally cut of the descriptions -- will fill out